partner_sector/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingenieria S.L. - Javier Iniesta
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    sector_id = fields.Many2one(comodel_name='res.partner.sector',
+                                string='Sector')
+
+    secondary_sector_ids = fields.Many2many(comodel_name='res.partner.sector',
```

Comment:
Domain should exclude sector found in `sector_id` to avoid having it in both primary and secondary sectors.

A onchange/constraint for it would be good too.